src/webui/www/private/views/about.html

```diff
@@ -26,48 +26,19 @@ <h3>QBT_TR(Current maintainer)QBT_TR[CONTEXT=AboutDialog]</h3>
         </tr>
     </table>
     </br>
-    <h3>QBT_TR(Original authors)QBT_TR[CONTEXT=HttpServer]</h3>
+    <h3>QBT_TR(Original author)QBT_TR[CONTEXT=HttpServer]</h3>
     <table>
         <tr>
             <td>QBT_TR(Name:)QBT_TR[CONTEXT=AboutDialog]</td>
-            <td>Ishan Arora</td>
+            <td>Christophe Dumez</td>
```

Comment:
IIRC, this page is changed to follow the one from GUI. Am I right?

Reply:
exactly.